shell2http.go

```diff
@@ -294,8 +294,11 @@ func getShellHandler(appConfig Config, path string, shell string, params []strin
 			shellOut, err = osExecCommand.Output()
 		}
 
-		if err != nil && !appConfig.showErrors {
+		if err != nil {
 			log.Println("exec error: ", err)
+		}
+
+		if err != nil && !appConfig.showErrors {
```

Comment:
I guess the same question as the one that lead to this change is worth asking: will a user now know, from looking at the `fmt` output, that an error occurred? Would it make sense to consider logic along the lines of:

```
if err != nil {
  fmt.Fprint(rw, "exec error: ", err)
}
if err == nil || appConfig.showErrors {
  outText := string(shellOut)
  ...
}
```
?